app/screens/custom_status/components/custom_status_input.tsx

```diff
@@ -42,11 +38,12 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
             paddingHorizontal: 16,
             textAlignVertical: 'center',
             height: '100%',
+            ...typography('Body', 200, 'Regular'),
```

Comment:
If you are using typography, you can remove the `fontSize`, since it is going to get overwritten either way.

Reply:
fixed, thanks for the catch!